src/dvc_studio_client/post_live_metrics.py

```diff
@@ -239,6 +241,10 @@ def post_live_metrics(  # noqa: C901,PLR0912,PLR0913
         if message:
             # Cutting the message to match the commit title length limit.
             body["message"] = message[:72]
+        if subdir:
+            body["subdir"] = subdir
+        if dvc_experiment_parent_data:
+            body["dvc_experiment_parent_data"] = dvc_experiment_parent_data
```

Comment:
minor / non-blocker:

I know that this name comes from Studio, but we might reconsider it for the API. E.g. we don't prefix other fields wit h `dvc_experiment_`

Reply:
In Studio it's a legacy naming because of different types of experiments

Reply:
I did consider this.

I decided that it was more important to have a consistent (and slightly annoying) name throughout the ~spider's web~ ecosystem. 

It has been my experience that cross-system renames (i.e. in APIs or directly before the DB) make it far more difficult to navigate the big picture.

We could pause and go back and rename the original field but I decided that was not worth the time commitment.

Reply:
Not critical / not a blocker. Yep, my concern is these names were not visible before (they are some internal impl details of Studio), and now we are kinda expanding the surface. My 2cs on this - I would not pay attention to any internal Studio-specific conventions in out public tools. And I think it's fine to migrate Studio later, even it's way later. It's more important to my mind for public tools / APIs to be clean.

Reply:
That would be where our opinion differs. I do not see `dvc-studio-client` as a public tool or API. I don't even think it should be a package that is external to DVC.

Reply:
It's public since it's part of the DVC project (DVC itself, or a separate package - in this case it doesn't really matter). Let's say if tomorrow someone (e.g. DagsHub, or GitLab) would want to implement a DVC experiments-compatible service they would be using this API. Or if some other tools decide to send metrics to Studio.

Or another scenario / angle. It is similar to let's say DVC sending metrics to MLFlow - it's a very reasonable scenario, and we'll be using their API to do this.

Reply:
While the way Studio operates internally (database schema, GraphQL schema, etc) - those are completely hidden and private and people don't care and won't care about those.

Reply:
> Let's say if tomorrow someone (e.g. DagsHub, or GitLab) would want to implement a DVC experiments-compatible service they would be using this API.

The chances of this hypothetical situation are much lower than some poor dev having to go back into this code to make changes.